fuzz/url.dict

```diff
@@ -0,0 +1,2 @@
+"https://"
```

Comment:
I think you should add all types of protocols (available as special schemes in url spec)

Reply:
Ok, I'll do it